.github/workflows/pre-commit.yaml

```diff
@@ -0,0 +1,21 @@
+name: pre-commit
+
+on:
+  pull_request:
+  push:
+    branches: [main]
+
+jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install Helm
+      uses: azure/setup-helm@v1
+      with:
+        version: v3.4.0
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Setup Python
+      uses: actions/setup-python@v2
+    - name: Run pre-commit
+      uses: pre-commit/action@v2.0.3
```

Comment:
Can it update PR? Or validate that the pre-commit was applied locally?

Reply:
Only validate locally, haven't tried updating PR with it yet.